Documentation/SpatialAwareness/ConfiguringSpatialAwarenessMeshObserver.md

```diff
@@ -1,123 +1,122 @@
-# Configuring the Spatial Awareness Mesh Observer
+# Configuring Mesh Observers for Device
 
-For every Spatial Mesh Observer entry in the Spatial Awareness System profile, the Mesh Observer Profile must define the type implementation and the list of supported platform(s) to run this observer.
+SDFSDFSDF
```

Comment:
Did you forget something? 😋

Reply:
Damn you guys were fast lol. Got that and docfx fixes